extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/model/NewMediaItemResult.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.datatransfer.google.photos.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class NewMediaItemResult {
```

Comment:
Can you get any of these classes from SDK?

Reply:
How does it avoid confusion (not disagreeing, just wondering)? I would either copy verbatim from SDK (and comment that it is copied), or ideally import from sdk so you don't have to copy.

Reply:
Looks like the SDK only has Java objects that work only with the Java methods that we can't, at present, use.  Using our own POJOs allows us to use the REST API.